Utility/integration-test.sh

```diff
@@ -16,7 +16,7 @@ mv Examples/Cocoa/Sources/Objective_C/*.h Examples/Cocoa/Sources/Objective_C/inc
 $PLANK_BIN --lang flow  --output_dir=Examples/JS/flow/ $JSON_FILES
 
 # Generate flow types for models
-$PLANK_BIN --lang java --java_package_name com.pinterest.models --output_dir=Examples/Java/Sources/ $JSON_FILES
+$PLANK_BIN --lang java --java_package_name com.pinterest.models --java_nullability_annotation_type androidx --output_dir=Examples/Java/Sources/ $JSON_FILES
```

Comment:
do you think it is worthwhile to test just the androidx option or should we have separate test fixtures for android-support as well?

Reply:
It feels a little bit overkill? Or maybe I could just test that both annotation types build, but only have one type committed as a test fixture. I would say it's probably ok to just test `androidx` going forward though, since this isn't a likely point of failure.